e2e_playwright/shared/app_utils.py

```diff
@@ -222,3 +222,40 @@ def click_checkbox(
     checkbox_element = get_checkbox(page, label)
     checkbox_element.click()
     wait_for_app_run(page)
+
+
+def expect_help_tooltip(
+    app: Page, el_with_help_tooltip: Locator, tooltip_text: str | Pattern[str]
```

Comment:
super nit: `el_with_help_tooltip` -> `element_with_help_tooltip` or `help_tooltip_element`

Reply:
Renamed to `element_with_help_tooltip`